libcontainer/rootfs_linux.go

```diff
@@ -188,10 +189,10 @@ func prepareTmp(topTmpDir string) (string, error) {
 		return "", err
 	}
 	if err := unix.Mount(tmpdir, tmpdir, "bind", unix.MS_BIND, ""); err != nil {
-		return "", err
+		return "", &os.PathError{Op: "bind-mount", Path: tmpdir, Err: err}
```

Comment:
Perhaps it makes sense to introduce a special `mountError`, similar to one from github.com/moby/sys/mount (unfortunately we can't use moby/sys/mount directly as it's too high-level).

Reply:
Done